### ISSUE TITLE
Container shares same history?

### ISSUE THREAD

**Issue by shahishasank** [mozilla/multi-account-containers#1826]
- Multi-Account Containers Version: 7.0.2
- Operating System + Version: Windows 8.1 64 bit
- Firefox Version: 78.0.2
- Other installed Add-ons + Version + Enabled/Disabled-Status: Random User Agent 2.2.13 Enabled

### Actual behavior
Links are already marked as clicked in multiple containers.

### Expected behavior
Every container should be fresh like a browser, there should be no history shared between containers.

### Steps to reproduce
1. Google anything
2. Click on link of any search result
3. Create new container
4. Repeat step 1 in new container

### Notes
You'll see the link is marked as clicked(purple color) in new container but since I created a new container it should be fresh like new. Please help
See ss for example
![image](https://user-images.githubusercontent.com/32937688/88041949-c592f000-cb68-11ea-8a35-fb5bc95d4de7.png)

![image](https://user-images.githubusercontent.com/32937688/88042258-481baf80-cb69-11ea-84a7-83f7f6755c04.png)

**Comment by groovecoder** [comment]
Note: this requires an upstream fix: https://bugzilla.mozilla.org/show_bug.cgi?id=1283320

**Comment by woj-tek** [comment]
That would be great way to separate "contexts". And what's more - easily clear history of certain containers.

**Comment by anewuser** [comment]
This is a duplicate of #952.

**Comment by julie777** [comment]
I disagree with this proposal. History is specific to a given tab. That history should be able to cross containers if the setting to "Replace tab instead of creating a new one" is enabled.
